packages/zsh.rb

```diff
@@ -3,33 +3,36 @@
 class Zsh < Package
   description 'Zsh is a shell designed for interactive use, although it is also a powerful scripting language.'
   homepage 'http://zsh.sourceforge.net/'
-  version '5.8.1-1'
+  @_ver = '5.9'
+  version @_ver
   license 'ZSH and GPL-2'
   compatibility 'all'
-  source_url 'https://downloads.sourceforge.net/project/zsh/zsh/5.8.1/zsh-5.8.1.tar.xz'
-  source_sha256 'b6973520bace600b4779200269b1e5d79e5f505ac4952058c11ad5bbf0dd9919'
+  source_url 'https://sourceforge.net/projects/zsh/files/zsh/5.9/zsh-5.9.tar.xz'
```

Comment:
If you are going to change the source url, you might as well get the final destination after all the redirects to `https://versaweb.dl.sourceforge.net/project/zsh/zsh/5.9/zsh-5.9.tar.xz`.  The above url actually adds an additional redirect.

Reply:
iirc sometimes the URL changes so u use the link I provided

Reply:
I've noticed there are 3 redirects with the url you provided.  I think the downloads url reduces that by at least one hop.

Reply:
I honestly didn't notice until now that the URI had changed from `https://downloads.sourceforge.net` to `https://sourceforge.net`. I changed it back to the old one. [69d7db5](https://github.com/skycocker/chromebrew/pull/7112/commits/69d7db5224dd97d34438d47136c096c82802093e)